### DISCUSSION TITLE
New name - New icon

### DISCUSSION THREAD

**Discussion by 0xMRTT** [GradienceTeam/Gradience#117] [General]
Recently, the name changed to Gradience.

Now, the icon is unrelated with the name

**Comment by daudix** [comment]
I stared sketching some variants

**Reply by daudix** [reply]
Any icon ideas are appreciated!

**Comment by 0xMRTT** [comment]
Maybe a color bucket

**Reply by 0xMRTT** [reply]
With a brush inside

**Reply by daudix** [reply]
Will try, Thanks!

**Reply by daudix** [reply]
This is tricky one in terms of shape, which is need to follow GNOME HIG

**Comment by 0xMRTT** [comment]
Gradience mean the absence of a clear-cut boundary between one category and another, for example between cup and mug in semantics. We can explore this way

**Reply by daudix** [reply]
Window (UI) and a window (real)?

**Reply by 0xMRTT** [reply]
Didn't understand 😅

**Reply by daudix** [reply]
UI window (like in current icon) with real life window inside, maybe with brush nearby

**Comment by daudix** [comment]
Have made a little control pannel, it can be added to current icon "window"

![control](https://user-images.githubusercontent.com/77155297/184451840-a2c1348e-3c29-4a1b-a595-cc26d8103ec4.png)

**Reply by 0xMRTT** [reply]
Nice

**Reply by daudix** [reply]
But not realted to the name :/

**Reply by 0xMRTT** [reply]
A little, because there is multiple color and the slider can be a gradient selector

**Comment by daudix** [comment]
![изображение](https://user-images.githubusercontent.com/77155297/184455748-c7a21ed0-5170-4335-ad7f-ce5981b947a6.png)

**Reply by 0xMRTT** [reply]
Nice 👍😎

**Reply by daudix** [reply]
## Blueprint Style

![изображение](https://user-images.githubusercontent.com/77155297/184455918-8939b5f0-fe7b-464e-a5d8-e8a3ba8fbe5f.png)

**Reply by 0xMRTT** [reply]
And what if the background is a gradient background?

**Reply by daudix** [reply]
![изображение](https://user-images.githubusercontent.com/77155297/184456247-6a8c3852-c19a-4947-bd96-61cfe6a06f0e.png)

**Comment by 0xMRTT** [comment]
All three seems to be great. In your opinion which one is better and which one follow the best HIG

**Reply by daudix** [reply]
Currently none of this, those are just a somthing quick before we will have an Amazing idea

**Reply by daudix** [reply]
And this is also a proof of that icon need to be new from scartch

**Comment by daudix** [comment]
## Maybe last try with this idea

![изображение](https://user-images.githubusercontent.com/77155297/184458352-ed6c4a09-cc44-4c91-89d0-e8bf7baeffbf.png)

**Comment by daudix** [comment]
![image](https://user-images.githubusercontent.com/77155297/184499440-136855aa-ad4c-4bd8-8b8b-c4621f89686b.png)

**Reply by 0xMRTT** [reply]
The "G" looks bad.

**Reply by 0xMRTT** [reply]
I like the idea of computer. Maybe adding a brush which is painting the computer

**Reply by daudix** [reply]
But again, not realted to the name...

**Reply by 0xMRTT** [reply]
Or color the screen with a gradient

**Reply by daudix** [reply]
![image](https://user-images.githubusercontent.com/77155297/184499771-de180527-fb5c-428c-89f2-6cab836955a7.png)

**Reply by 0xMRTT** [reply]
Maybe the same, but replace the rainbow by a brush

**Comment by daudix** [comment]
![image](https://user-images.githubusercontent.com/77155297/184499520-9db2674f-3ca6-4a2e-ade5-f0fd68b749d0.png)

**Reply by daudix** [reply]
I know, this is bad one

**Reply by 0xMRTT** [reply]
Don't worry

**Comment by daudix** [comment]
Honestly, I have no idea for a good icon, symbolic for matrix, and etc

**Reply by 0xMRTT** [reply]
We can keep the current one until we found an amazing icon

**Reply by daudix** [reply]
Hopefully we will find it before v0.2.0

**Comment by daudix** [comment]
![image](https://user-images.githubusercontent.com/77155297/184503800-d71988a2-5276-418d-9bab-2a5c0530d69e.png)

**Reply by ZechariahB** [reply]
Narrator: "Perhaps it was some sort of birthday party the others were holding. Maybe this was why there is not a single soul within this building", Stanley thought to himself. Now all alone, he decided to return to his office to get back to work. He enters through the door on the left.

Narrator: Wrong door, Stanley! What are you doing!? This door I especially made for you was supposed to have yourself enlightened by my wonderfully crafted narrative. We were going to have an grand time together like the splendid good old days of 2013, even better than when we followed the adventure line. I decorated it myself to make it ever more so enticing to enter. That should say a lot I do say so myself. We should get back on track immediately to not miss out on this wonderful opportunity to try something new. 

Narrator: Stanley took a detour back his office and looked at his computer to discover... "The Stanley Parable Reassurance Bucket" is now an app! You got to trust me on this, Stanley. Every time we took the bucket it was always promising. We never know what exciting things will happen if we click it.

**Reply by daudix** [reply]
Dude, I really made this but though no one will understand )))

_Yup, back then bucket was unfinished..._

![Снимок экрана от 2022-08-13 18-47-48](https://user-images.githubusercontent.com/77155297/184559453-98477954-10a6-4168-876c-caba424f7f38.png)

**Comment by 0xMRTT** [comment]
![pepper](https://user-images.githubusercontent.com/105598867/184503854-a0ff12c7-b325-440d-9a07-077bd348f23c.jpg)

**Reply by daudix** [reply]
???

**Reply by daudix** [reply]
A Pepper... But not GIMP one... Hmm...

**Reply by 0xMRTT** [reply]
It's what i got when typing `Gradience`

**Reply by daudix** [reply]
This is M/F peppers, Thank you for making me understand this termin

How how this is realted to UI recoloring app?

**Comment by daudix** [comment]
![image](https://user-images.githubusercontent.com/77155297/184504501-6a7d41e9-a210-4a83-ab0a-3e52bc1ac1fb.png)

**Comment by daudix** [comment]
Maybe just keep current one?

_"Paint Roller repaints a normal Adwaita window and makes is look a bit different, but it still work the same"_

**Reply by 0xMRTT** [reply]
Yes 👍🙌👍🙌🙌🙌 👍👍

**Reply by 0xMRTT** [reply]
You mean changing the name to paint roller?

**Reply by daudix** [reply]
Not really

**Reply by daudix** [reply]
Just meaning of the icon

**Reply by 0xMRTT** [reply]
Okay. It was what I was thinking 🙂.
